src/Simplex/Chat/Migrations/M20230511_reactions.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20230511_reactions where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20230511_reactions :: Query
+m20230511_reactions =
+  [sql|
+CREATE TABLE chat_item_reactions (
+  chat_item_reaction_id INTEGER PRIMARY KEY AUTOINCREMENT,
+  item_member_id BLOB, -- member that created item, NULL for items in direct chats
+  shared_msg_id INTEGER NOT NULL,
+  contact_id INTEGER REFERENCES contacts ON DELETE CASCADE,
+  group_id INTEGER REFERENCES groups ON DELETE CASCADE,
+  group_member_id INTEGER REFERENCES group_members ON DELETE SET NULL, -- member that sent reaction, NULL for items in direct chats
+  created_by_msg_id INTEGER UNIQUE REFERENCES messages(message_id) ON DELETE SET NULL,
```

Comment:
maybe not make it unique? it could be allowed to create several reactions via single message